packages/dotfiles/dot_claude/skills/apple-hig-helper/process-to-markdown.py

```
@@ -1,244 +1,244 @@
+            result = process_file_simple(html_file, url, output_file)
+
+        if result:
+            success_count += 1
+        else:
+            fail_count += 1
+
+        # Force garbage collection every 10 files to reduce memory usage
+        if (i + 1) % 10 == 0:
+            gc.collect()
+
+    print(f"\n=== Summary ===")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)